### DISCUSSION TITLE
Prezentácie

### DISCUSSION THREAD

**Discussion by frankhorv** [Rust-Slovakia/Bratislava-Rust-Meetup#3] [Ideas]
Mas zaujem odprezentovat svoj napad?
Je nejake tema, o ktorej by si sa chcel porozpravat v sirsej skupine?

**Comment by Kixunil** [comment]
Mozem si pripravit tieto temy (EN/SK podla situacie):

Declarative programming in Rust - overview of the libraries: serde, configure_me, protobuf/gRPC, ...
Testing Rust code - approaches and libraries
Rust patterns - solutions for common problems in Rust
Async - how it works, why/when to use it and how to use it?
All about error handling
Unsafe Rust - misconceptions, footguns, patterns

**Comment by frankhorv** [comment]
@Kixunil 28.6. planujeme dalsi Meetup, vedel by si si pripravit nejake temy na prezentaciu/diskusiu?
